<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cChapterWindow</class>
 <widget class="QWidget" name="cChapterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[chapter] - </string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,5,10">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="cTextEdit" name="m_lpText"/>
     </item>
     <item row="1" column="1">
      <widget class="cLineEdit" name="m_lpName"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Scenes:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="cTextEdit" name="m_lpDescription"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="cTreeView" name="m_lpSceneList">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Part:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="cLineEdit" name="m_lpPart">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ctextedit.h</header>
  </customwidget>
  <customwidget>
   <class>cLineEdit</class>
   <extends>QLineEdit</extends>
   <header>clineedit.h</header>
  </customwidget>
  <customwidget>
   <class>cTreeView</class>
   <extends>QTreeView</extends>
   <header>ctreeview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_lpName</tabstop>
  <tabstop>m_lpDescription</tabstop>
  <tabstop>m_lpSceneList</tabstop>
  <tabstop>m_lpText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
